core/src/main/java/bisq/core/offer/OfferUtil.java

```diff
@@ -306,7 +321,9 @@ static Coin getAdjustedAmount(Coin amount, Price price, long maxTradeLimit, int
         }
     }
 
-    public static String getFeeWithFiatAmount(Coin makerFeeAsCoin, Optional<Volume> optionalFeeInFiat, BSFormatter formatter) {
+    public static String getFeeWithFiatAmount(Coin makerFeeAsCoin,
+                                              Optional<Volume> optionalFeeInFiat,
+                                              BSFormatter formatter) {
```

Comment:
please only change stuff that actually changes something. Makes it cumbersome to review...